tests/v2/validation/rbac/globalrolesv2/globalroles_v2.go

```diff
@@ -0,0 +1,196 @@
+			return grbs.Name, nil
+		}
+	}
+	return "", nil
+}
+
+func listClusterRoleTemplateBindingsForInheritedClusterRoles(client *rancher.Client, grbOwner string, expectedCount int) (*v3.ClusterRoleTemplateBindingList, error) {
+	req, err := labels.NewRequirement(crtbOwnerLabel, selection.In, []string{grbOwner})
+	if err != nil {
+		return nil, err
+	}
+	selector := labels.NewSelector().Add(*req)
+
+	var crtbs *v3.ClusterRoleTemplateBindingList
+
+	err = kwait.Poll(500*time.Millisecond, 60*time.Second, func() (done bool, pollErr error) {
+		crtbs, pollErr = rbac.ListClusterRoleTemplateBindings(client, metav1.ListOptions{
+			LabelSelector: selector.String(),
+		})
+		if pollErr != nil {
+			return false, pollErr
+		}
+		if len(crtbs.Items) == expectedCount {
+			return true, nil
+		}
+		return false, nil
+	})
+
+	if err != nil {
+		return nil, err
+	}
+	return crtbs, nil
+}
+
+func getCRBsForCRTBs(client *rancher.Client, crtbs *v3.ClusterRoleTemplateBindingList) (*rbacv1.ClusterRoleBindingList, error) {
+	var downstreamCRBs rbacv1.ClusterRoleBindingList
+	for _, crtb := range crtbs.Items {
+		labelKey := fmt.Sprintf("%s_%s", crtb.ClusterName, crtb.Name)
+		req, err := labels.NewRequirement(labelKey, selection.In, []string{bindingLabel})
+		if err != nil {
+			return nil, err
+		}
+
+		selector := labels.NewSelector().Add(*req)
+		downstreamCRBsForCRTB, err := rbac.ListClusterRoleBindings(client, localcluster, metav1.ListOptions{
+			LabelSelector: selector.String(),
+		})
+		if err != nil {
+			return nil, err
+		}
+		downstreamCRBs.Items = append(downstreamCRBs.Items, downstreamCRBsForCRTB.Items...)
+	}
+	return &downstreamCRBs, nil
+}
+
+func getRBsForCRTBs(client *rancher.Client, crtbs *v3.ClusterRoleTemplateBindingList) (*rbacv1.RoleBindingList, error) {
+	var downstreamRBs rbacv1.RoleBindingList
+	for _, crtb := range crtbs.Items {
+		roleTemplateName := crtb.RoleTemplateName
+		if strings.Contains(roleTemplateName, "rt") {
+			listOpt := metav1.ListOptions{
+				FieldSelector: "metadata.name=" + roleTemplateName,
+			}
+			roleTemplateList, err := rbac.ListRoleTemplates(client, listOpt)
+			if err != nil {
+				return nil, err
+			}
+			roleTemplateName = roleTemplateList.Items[0].RoleTemplateNames[0]
+
+		}
+		nameSelector := fmt.Sprintf("metadata.name=%s-%s", crtb.Name, roleTemplateName)
+		namespaceSelector := fmt.Sprintf("metadata.namespace=%s", crtb.ClusterName)
+		combinedSelector := fmt.Sprintf("%s,%s", nameSelector, namespaceSelector)
+
+		downstreamRBsForCRTB, err := rbac.ListRoleBindings(client, localcluster, "", metav1.ListOptions{
+			FieldSelector: combinedSelector,
+		})
+		if err != nil {
+			return nil, err
+		}
+		downstreamRBs.Items = append(downstreamRBs.Items, downstreamRBsForCRTB.Items...)
+	}
+	return &downstreamRBs, nil
+}
+
+func createDownstreamCluster(client *rancher.Client, clusterType string) (*management.Cluster, *v1.SteveAPIObject, *clusters.ClusterConfig, error) {
```

Comment:
we are trying to move away from creating clusters as part of tests that aren't exclusively testing provisioning. Is there a way this can be done using a user-provided cluster via the 'cluster-name' yaml variable?

Reply:
What we can also do is move this specific test case into the provisioning package. I guess what makes the most sense? Would it make sense to add to the table in our existing provisioning tests?

Reply:
After thinking about this a bit more, Priya is even necessary to actually create a cluster? Wouldn't it make more sense to just check if it's allowed ie checking the schema? Doing something like this? 
https://github.com/rancher/rancher/blob/5b42ca50475398112661c80ac2b9070d5bacfb24/tests/framework/pkg/clientbase/ops.go#L190-L197

Reply:
Israel, the focus of these tests extends a little beyond just assessing the user's capability to create a cluster. We are also validating that when a new cluster is created by the user with the inherited cluster role, the respective CRTBs, CRBs and RBs are created as well. This is key in these tests. Additionally, we will be performing cluster-owner checks where we create projects/namespaces within the cluster. This is yet to be implemented (https://github.com/rancher/qa-tasks/issues/1038) after Anu's refactoring RBAC PR is merged. 
Is it a hard requirement to not have cluster creation in tests other than provisioning tests?

Reply:
Ah I see, I mean we would like to, but I also understand that tests will cross over to other areas hmmm ... @slickwarren @markusewalker any thoughts? Priya makes a great point, but would like your guys' input as well since you are the provisioning team.

Reply:
As an addition to the note above, would it make sense to make provisioning tests based on these roles, and validate that the CRTBs, CRBs and RBs are created?  Please let me know your thoughts.

Reply:
These tests are primarily centered on RBAC, ensuring the creation of specific RBAC resources. I'm hesitant about transferring them to the provisioning package. Moving them could lead to significant code duplication, especially given the numerous RBAC-specific private helper functions we already have in the rbac and globalrolesv2 packages.

Reply:
Maybe we can have an adhoc meeting offline regarding this? Might be easier to sync up and figure out best scenario to work out.

Reply:
Sounds good, that way we can come to a consensus. Priya can you set up a meeting?

Reply:
Sure, will do. Since Anu is unavailable this week and Israel next week, I can set up a meeting for 11/27. However, should we delay this PR until after our meeting, OR is it acceptable to proceed as it is currently and make changes based on the outcome of our discussion later?

Reply:
I agree, we already have a test like this in rbac. This way we can come up with a consensus for these tests, the rbac test, and future tests. If anybody disagrees let me know

Reply:
Thanks Israel! For the time being, I will keep these tests involving new cluster provisioning as is. Based on the outcome of our meeting, we can later create a qa task to update the tests.